File: 06 Custom Properties/properties/SolidWorksMacro.vb
Imports SolidWorks.Interop.sldworks
Imports SolidWorks.Interop.swconst
Imports System.Runtime.InteropServices
Imports System


Partial Class SolidWorksMacro

    Public Sub main()
        LoadForm(swApp)
    End Sub



    ''' <summary>
    ''' The SldWorks swApp variable is pre-assigned for you.
    ''' </summary>
    Public swApp As SldWorks
End Class

File: C# Examples/06 Custom Properties/properties/SolidWorksMacro.cs
using SolidWorks.Interop.sldworks;

namespace properties
{
    public partial class SolidWorksMacro
    {
        public void Main()
        {

            
            Class1.LoadForm(swApp);

            return;
        }

        // The SldWorks swApp variable is pre-assigned for you.
        public SldWorks swApp;

    }
}

